api/part1/standard/sections/clause_10_requirements_class_deployment_features.adoc

```diff
@@ -73,9 +73,10 @@ NOTE: For deployments of models or processes, the location would usually not be
 | `controlstreams`      | - | The `Control Streams` that received commands issued during the `Deployment`. | A list of `ControlStream` resources (inline or by-reference). | Optional
 |====
 
-NOTE: The platform can be another `System` (e.g. a UAV or a station), but can also be any feature (e.g. a building or a tree).
-
+When referencing a deployed local system resource via `deployedSystems`, the system SHALL be identified using a URN.
+When referencing an external system resource the system SHALL be identified using a URL.
```

Comment:
Since this is somewhat encoding specific, I think it should be specified in the encoding sections instead:
- For GeoJSON, add a requirement below the [association mapping table](https://docs.ogc.org/DRAFTS/23-001r0.html#deployment-assoc-geojson-mappings) 
- For SensorML-JSON, add a requirement below the [association mapping table](https://docs.ogc.org/DRAFTS/23-001r0.html#deployment-assoc-sml-mappings) 

In the requirements, we should refer to the System `uniqueIdentifier` instead of just saying "a URN".

Also, if you look at Deployment examples in [GeoJSON](https://docs.ogc.org/DRAFTS/23-001r0.html#example_geojson_dep1) and [SensorML-JSON](https://docs.ogc.org/DRAFTS/23-001r0.html#example_sensorml_dep1), you will see that both `href` and `uid` can be provided in the schema. So I think when you read it back, the `uid` property already contains the URN and the href can remain the URL. However, what I had in mind (sorry I did not express this clearly in the issue), is that on insertion, to simplify things for the server, we could mandate that the href is the UID instead of the URL when linking to a local systems (whereas for remote systems, either the href URL only, or both URL and UID would be provided).

Does this make sense?